modules/tests/jvm/src/main/scala/gsp/math/geom/jts/demo/GmosScienceAreaGeometry.scala

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gsp.math.geom.jts.demo
+
+import gsp.math.{ Angle, Offset }
+import gsp.math.geom._
+import gsp.math.geom.syntax.all._
+
+import gsp.math.syntax.int._
+
+/**
+ * GMOS science area geometry.
+ */
+object GmosScienceAreaGeometry {
+
+  val imaging: ShapeExpression =
+    imagingFov(330340.mas, 33840.mas)
```

Comment:
Where do these numbers come from? Are they sepecified in some documento or come from the old codebase?

Reply:
Unfortunately I don't have a great answer.  I took them from the old codebase and I'm not sure where to find GMOS documentation for this.